### PR TITLE
[ZEPPELIN-954] Fix table cell selection problem on second run by properly destroying hot.

### PR DESCRIPTION
**Pull request by jasonxh**
### What is this PR for?
* Fix table cell selection problem on second run by properly destroying hot.
* Also make cells readonly. Previously one were able to paste into them.

### What type of PR is it?
[Bug Fix]

### What is the Jira issue?
* [ZEPPELIN-954]

### How should this be tested?
Execute the following paragraph multiple times, and verify the table cells are still selectable.
```
%sh
echo %table
echo -e "col1\tcol2\tcol3"
echo -e "1\t2.1\tabcdefg"
```
Also try to paste anything into a cell to no avail.

### Questions:
* Does the licenses files need update? No
* Is there breaking changes for older versions? No
* Does this needs documentation? No

### CODE REVIEW

**Comment by minahlee** [comment]
Tested both cases, looks good to me 👍

**Comment by minahlee** [comment]
\cc @corneadoug for review

File: zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js [hunk 1]
```diff
@@ -1224,12 +1224,16 @@ angular.module('zeppelinWebApp')
   var setTable = function(type, data, refresh) {
     var renderTable = function() {
       var height = $scope.paragraph.config.graph.height;
-      angular.element('#p' + $scope.paragraph.id + '_table').css('height', height);
+      var container = angular.element('#p' + $scope.paragraph.id + '_table').css('height', height).get(0);
```
Comment (corneadoug) [inline zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js]: Tested, works great, but could we keep those two actions separated?

**Comment by corneadoug** [comment]
Tested, No more handsontable duplicates.
Was there no other way to just update the data instead of destroying the table and creating a new one?

File: zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js (hunk 1 above)
Comment (jasonxh) [inline zeppelin-web/src/app/notebook/paragraph/paragraph.controller.js]: i figured we could just start using the return value and save an extra lookup for the same selector. pls let me know if you have strong preference and i can revert this line.

**Comment by jasonxh** [comment]
@corneadoug good suggestion. i updated the pr to reuse the table when possible.

**Comment by minahlee** [comment]
Merge it there is no more discussion

**Pull request closed without merging**